msdsn/conn_str.go

```diff
@@ -117,7 +118,7 @@ func Parse(dsn string) (Config, map[string]string, error) {
 			return p, params, err
 		}
 		params = parameters
-	} else if strings.HasPrefix(dsn, "sqlserver://") {
+	} else if strings.HasPrefix(dsn, "sqlserver://") || strings.HasPrefix(dsn, "azuresql://") {
```

Comment:
If we do this, let's create a global regexp:
`^[a-zA-Z]+:\/\/`

To test for this.